theano/tensor/elemwise.py

```diff
@@ -125,6 +125,7 @@ class DimShuffle(Op):
 
     _f16_ok = True
     check_input = False
+    __props__ = ("input_broadcastable", "new_order")
```

Comment:
I missed it, but inplace should be in this list. But this will cause problem on the GPU as it don't exist on GPU. The real solution would be to remove the inplace parameter and suppose it is always True. Then we will need to remove the opt dimshuffle_as_view and local_dimshuffle_no_inplace_at_canonicalize. Also, in the file tensor/basic.py, in the function transpose, remove the inplace parameter.

Maybe there is other clean up. So run the tests locally if possible.

thanks